OpenTrons/opentrons-api
# fix(app): apply identity vector labware offsets

closes [RQA-1626](https://opentrons.atlassian.net/browse/RQA-1626)
closes [RQA-1627](https://opentrons.atlassian.net/browse/RQA-1627)

# Overview

Currently, identity vector labware offset data are filtered in the CurrentOffsetsTable component. This unintentionally renders and empty offsets table when there are some offsets saved and they are all equal to the identity vector, rather than the intended "No labware offset data yet" text.  

This PR also takes care of a minor styling bug in the table header when there are some offsets.

# Test Plan

- Verify that if no offset data is saved, "No labware offset data available" renders on the choose protocol/robot slideout.
- Verify that if no offset data is saved, SetupLabwarePositionCheck component renders "No labware offset data yet" rather than an empty table.
- Verify that if only identity offset data is saved, "No labware offset data available" renders on the choose protocol/robot slideout.
- Verify that if only identity offset data is saved, SetupLabwarePositionCheck component renders "No labware offset data yet" rather than an empty table.
- Verify that if some identity offset data and some non-identity offset data is saved, "Apply offset data" renders on the choose protocol/robot slideout. Selecting "View data" should show a table of only non-identity offsets.
- Verify that if some identity offset data and some non-identity offset data is saved, SetupLabwarePositionCheck component renders a CurrentOffsetsTable component showing only non-identity offsets.


[RQA-1626]: https://opentrons.atlassian.net/browse/RQA-1626?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RQA-1627]: https://opentrons.atlassian.net/browse/RQA-1627?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
